ls1intum/Artemis
# [Exam] Add second table with assessed submissions from other tutors

### Checklist
- [x] We checked thoroughly that the conventional exercise assessments still work (Save, Submit, Continue, Override, ...)
- [x] We tested the exam assessment multiple times with all important exercise types and at least three tutors assessing concurrently

### Motivation and Context
The exam correction entails a second assessment by a second tutor. We want to enable tutors to double-check an assessment.

### Description
- [x] In the tutor dashboard, display the already assessed submissions from other tutors. This table is only shown for exam exercises.
- [x] Provide an action button to edit the existing assessment. (client)
- [x] Allow Tutors to edit assessments of exam exercises (Server)


### Steps for Testing
Firstly, make sure that no functionality of the tutor dashboard has been broken. All changes should have no effect on normal exercises. 

1. Log in to Artemis
2. Create an exam and create exercises (text & programming).
3. Start the exam as a student and submit. 
4. Log in as a tutor and assess the exercises from the tutor dashboard. 
5. Log in as another tutor. In the tutor dashboard, the already assessed submission should be shown in the corresponding table

### Screenshots
**Tutor 8:**
![image](https://user-images.githubusercontent.com/15110960/87222437-f35f8400-c373-11ea-869e-f7beb520ea03.png)

**Tutor 10:**
<img width="1182" alt="image" src="https://user-images.githubusercontent.com/41108487/87222450-05d9bd80-c374-11ea-828c-0ca24d076079.png">



**After second correction:**
**Tutor 8:**
![image](https://user-images.githubusercontent.com/15110960/87222501-4a655900-c374-11ea-880a-497b6b47bad0.png)

**Tutor 10:**
<img width="1172" alt="image" src="https://user-images.githubusercontent.com/41108487/87222496-3e799700-c374-11ea-994f-2adf76f0aa17.png">
